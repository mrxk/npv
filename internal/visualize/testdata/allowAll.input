---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: network-policy
  namespace: default
spec:
  podSelector:
    matchLabels:
      app: demo
  policyTypes:
    - Ingress
    - Egress
  ingress:
    - {}
  egress:
    - {}
