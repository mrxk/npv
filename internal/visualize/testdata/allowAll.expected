@startuml
left to right direction
frame Pods {
component "Name: network-policy\lNamespace: default\lMatch Labels:\l    app: demo\l" as defaultappdemo {
    port "0-65535" as _ALL_PEER_port
    portout " " as defaultappdemoportout
}
}
frame Ingress {
component "ALL" as _ALL_PEER_INGRESS {
    portout " " as _ALL_PEER_INGRESSingressportout
}
}
_ALL_PEER_INGRESSingressportout --down[#green]--> _ALL_PEER_port
frame Egress {
component "ALL" as _ALL_PEER_ {
    port "0-65535" as _ALL_PEER_egressport
}
}
defaultappdemoportout --down[#green]--> _ALL_PEER_egressport
@enduml
